packages/Pyms/Pyms.0.2/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+synopsis: "OCaml typedefinition to python3 transpiler"
+description: """
+Pyms is a transpiler. It produces clean and reliable python modules from OCaml type definitions.
+
+More than a transpiler, Pyms is also a language extension built on top of python allowing you to define OCaml-like variants and use them from a classical python source code.
+"""
+maintainer: "ARTHUR CORRENSON arthur.correnson@univ-tlse3.fr"
+authors: "ARTHUR CORRENSON arthur.correnson@univ-tlse3.fr"
+license: "MIT"
+homepage: "https://github.com/jdrprod/Pym-s"
+bug-reports: "https://github.com/jdrprod/Pym-s/issues"
+dev-repo: "git+https://github.com/jdrprod/Pym-s.git"
+depends: [ "ocaml" "ocamlfind" "dune"]
```

Comment:
```suggestion
depends: [ "ocaml" "dune" {>= "1.9.0"} "menhir" "ocaml-compiler-libs"]
```
From a brief look at the code, I think you got the dependencies wrong. The CI will likely point out additional missing bounds